pexpect/pty_spawn.py

```diff
@@ -677,11 +677,12 @@ def interact(self, escape_character=chr(29),
         the stdout and stderr output of the child process is printed. This
         simply echos the child stdout and child stderr to the real stdout and
         it echos the real stdin to the child stdin. When the user types the
-        escape_character this method will stop. The default for
-        escape_character is ^]. This should not be confused with ASCII 27 --
-        the ESC character. ASCII 29 was chosen for historical merit because
-        this is the character used by 'telnet' as the escape character. The
-        escape_character will not be sent to the child process.
+        escape_character this method will stop. Escaping may be disableall-together
```

Comment:
I realize my quick draft resulted in this, but please: ``"disableall-together" => "disabled entirely"``